pkgs/by-name/jj/jjui/package.nix

```
@@ -0,0 +1,34 @@
+{
+  pkgs,
+  lib,
+  ...
+}:
+pkgs.buildGoModule rec {
+  pname = "jjui";
+  version = "b1f467fcef89a6f9e57a5be2108bb0ef34f1442d";
+
+  src = pkgs.fetchFromGitHub {
+    owner = "idursun";
+    repo = pname;
```

```suggestion
    repo = "jjui";
```
See https://github.com/NixOS/nixpkgs/issues/277994

---

Fair enough. Changed. Thanks for the context.